tests/test_homo_sapiens.py

```diff
@@ -26,33 +26,42 @@ def test_str(self):
-        self.assertEqual(genome.chromosomes["chr7"].length, 159345973)
-        self.assertEqual(genome.chromosomes["chr8"].length, 145138636)
-        self.assertEqual(genome.chromosomes["chr9"].length, 138394717)
-        self.assertEqual(genome.chromosomes["chr10"].length, 133797422)
-        self.assertEqual(genome.chromosomes["chr11"].length, 135086622)
-        self.assertEqual(genome.chromosomes["chr12"].length, 133275309)
-        self.assertEqual(genome.chromosomes["chr13"].length, 114364328)
-        self.assertEqual(genome.chromosomes["chr14"].length, 107043718)
-        self.assertEqual(genome.chromosomes["chr15"].length, 101991189)
-        self.assertEqual(genome.chromosomes["chr16"].length, 90338345)
-        self.assertEqual(genome.chromosomes["chr17"].length, 83257441)
-        self.assertEqual(genome.chromosomes["chr18"].length, 80373285)
-        self.assertEqual(genome.chromosomes["chr19"].length, 58617616)
-        self.assertEqual(genome.chromosomes["chr20"].length, 64444167)
-        self.assertEqual(genome.chromosomes["chr21"].length, 46709983)
-        self.assertEqual(genome.chromosomes["chr22"].length, 50818468)
-        self.assertEqual(genome.chromosomes["chrX"].length, 156040895)
+        self.assertEqual(genome.chromosomes["chr1"].length, 249218992)
+        self.assertEqual(genome.chromosomes["chr2"].length, 243090997)
+        self.assertEqual(genome.chromosomes["chr3"].length, 197874528)
+        self.assertEqual(genome.chromosomes["chr4"].length, 191028645)
+        self.assertEqual(genome.chromosomes["chr5"].length, 180715810)
+        self.assertEqual(genome.chromosomes["chr6"].length, 171051316)
+        self.assertEqual(genome.chromosomes["chr7"].length, 159126774)
+        self.assertEqual(genome.chromosomes["chr8"].length, 146301156)
+        self.assertEqual(genome.chromosomes["chr9"].length, 141109979)
+        self.assertEqual(genome.chromosomes["chr10"].length, 135503693)
+        self.assertEqual(genome.chromosomes["chr11"].length, 134945522)
+        self.assertEqual(genome.chromosomes["chr12"].length, 133778796)
+        self.assertEqual(genome.chromosomes["chr13"].length, 115106996)
+        self.assertEqual(genome.chromosomes["chr14"].length, 107288377)
+        self.assertEqual(genome.chromosomes["chr15"].length, 102506108)
+        self.assertEqual(genome.chromosomes["chr16"].length, 90163588)
+        self.assertEqual(genome.chromosomes["chr17"].length, 81045269)
+        self.assertEqual(genome.chromosomes["chr18"].length, 78015180)
+        self.assertEqual(genome.chromosomes["chr19"].length, 59097160)
+        self.assertEqual(genome.chromosomes["chr20"].length, 62949445)
+        self.assertEqual(genome.chromosomes["chr21"].length, 48100155)
+        self.assertEqual(genome.chromosomes["chr22"].length, 51229805)
+        self.assertEqual(genome.chromosomes["chrX"].length, 154929412)
         self.assertEqual(genome.chromosomes["chrY"].length, 57227415)
```

Comment:
Similar tests for recombination rate?

Reply:
done

Reply:
Sorry for my ignorance here, but why are such tests needed? Aren't you're just as likely to accidentally change a hard coded value in a test like this as you are the underlying file?

Reply:
I was wondering that as well tbh. maybe because we want to make sure no added code accidentally modifies these values. In other words, we always want these values to be fixed and the tests will make sure they are.

Reply:
Yeah, it's arguable. You have to get it wrong the same way twice with these tests though. Personally I'm a bit dyslexic, so I don't trust myself to get this sort of thing right.